packages/tradershub/src/components/Modal/Modal.tsx

```diff
@@ -47,7 +46,7 @@ const Modal = ({ children, className }: TModal) => {
     return (
         <div
             className={qtMerge(
-                'flex flex-col h-screen w-screen bg-background-primary-base lg:mx-auto lg:h-full lg:w-full lg:rounded-400',
+                'flex flex-col h-screen w-screen bg-system-light-primary-background lg:mx-auto lg:h-full lg:w-full lg:rounded-400',
```

Comment:
@sergei-deriv Why did you change this color?

Reply:
@mahdiyeh-deriv because base background for modal is white, not gray. All modals I've seen had white background